static/js/redux/init.jsx

```diff
@@ -1,204 +1,205 @@
-/*

```

Comment:
Why is this showing as entire file change?

Reply:
this has been happening every time someone alters `init.jsx` (see jeanie's PR https://github.com/jhuopensource/semesterly/pull/152/files#diff-51b3d0226cab8b1f22378afdccd04fd620de498f2d4c0745d4357340dee1c04a) - I am not sure why. At first I thought that maybe it was a difference in spacing btw his IDE and here but I'm not sure why this would only affect this file?